<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OUTPUT HASIL</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>951</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel_canny">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel_bilateral">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel_gray">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Grayscale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bilateral Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Canny Edge</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>70</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="KontrasLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kecerahan Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deteksiButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deteksi Plat</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutputLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="action_Grayscale"/>
    <addaction name="actionOperasi_Pencerahan"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Stretching"/>
    <addaction name="actionNegativeImage"/>
    <addaction name="actionBiner_Image"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuReset">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuReset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOperasi_Pencerahan">
   <property name="text">
    <string>Brightness</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionNegative_Image">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="actionNegativeImage">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_Grayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_Grayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_RGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="actionMin90_Derajat">
   <property name="text">
    <string>-90 Derajat</string>
   </property>
  </action>
  <action name="actionMin45_Derajat">
   <property name="text">
    <string>-45 Derajat</string>
   </property>
  </action>
  <action name="action45_Derajat">
   <property name="text">
    <string>45 Derajat</string>
   </property>
  </action>
  <action name="action90_Derajat">
   <property name="text">
    <string>90 Derajat</string>
   </property>
  </action>
  <action name="action180_Derajat">
   <property name="text">
    <string>180 Derajat</string>
   </property>
  </action>
  <action name="actionSkala_Dimensi">
   <property name="text">
    <string>Skewed Image</string>
   </property>
  </action>
  <action name="actionZoomIn_3x">
   <property name="text">
    <string>Zoom In (3x)</string>
   </property>
  </action>
  <action name="actionZoomIn_4x">
   <property name="text">
    <string>Zoom In (4x)</string>
   </property>
  </action>
  <action name="actionZoomOut14x">
   <property name="text">
    <string>Zoom Out (1/4x)</string>
   </property>
  </action>
  <action name="actionZoomOut34x">
   <property name="text">
    <string>Zoom Out (3/4x)</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionTambah">
   <property name="text">
    <string>Tambah</string>
   </property>
  </action>
  <action name="actionOperasi_AND">
   <property name="text">
    <string>Operasi AND</string>
   </property>
  </action>
  <action name="actionOperasi_OR">
   <property name="text">
    <string>Operasi OR</string>
   </property>
  </action>
  <action name="actionOperasi_XOR">
   <property name="text">
    <string>Operasi XOR</string>
   </property>
  </action>
  <action name="actionKurang">
   <property name="text">
    <string>Kurang</string>
   </property>
  </action>
  <action name="actionKali">
   <property name="text">
    <string>Kali</string>
   </property>
  </action>
  <action name="actionBagi">
   <property name="text">
    <string>Bagi</string>
   </property>
  </action>
  <action name="actionTranspose">
   <property name="text">
    <string>Transpose</string>
   </property>
  </action>
  <action name="action2x">
   <property name="text">
    <string>2x</string>
   </property>
  </action>
  <action name="action3x">
   <property name="text">
    <string>3x</string>
   </property>
  </action>
  <action name="action4x">
   <property name="text">
    <string>4x</string>
   </property>
  </action>
  <action name="action1_2x">
   <property name="text">
    <string>1/2x</string>
   </property>
  </action>
  <action name="action1_4x">
   <property name="text">
    <string>1/4x</string>
   </property>
  </action>
  <action name="action3_4x">
   <property name="text">
    <string>3/4x</string>
   </property>
  </action>
  <action name="filterKernel_1">
   <property name="text">
    <string>Kernel 1</string>
   </property>
  </action>
  <action name="filterKernel_2">
   <property name="text">
    <string>Kernel 2</string>
   </property>
  </action>
  <action name="meanKernel_1">
   <property name="text">
    <string>Kernel 1</string>
   </property>
  </action>
  <action name="meanKernel_2">
   <property name="text">
    <string>Kernel 2</string>
   </property>
  </action>
  <action name="actionGaussian_Filter">
   <property name="text">
    <string>Gaussian Filter</string>
   </property>
  </action>
  <action name="actionSharp_i">
   <property name="text">
    <string>Kernel i</string>
   </property>
  </action>
  <action name="actionSharp_ii">
   <property name="text">
    <string>Kernel ii</string>
   </property>
  </action>
  <action name="actionSharp_iii">
   <property name="text">
    <string>Kernel iii</string>
   </property>
  </action>
  <action name="actionSharp_iv">
   <property name="text">
    <string>Kernel iv</string>
   </property>
  </action>
  <action name="actionSharp_v">
   <property name="text">
    <string>Kernel v</string>
   </property>
  </action>
  <action name="actionSharp_vi">
   <property name="text">
    <string>Kernel vi</string>
   </property>
  </action>
  <action name="actionFilter_Laplace">
   <property name="text">
    <string>Kernel Laplace</string>
   </property>
  </action>
  <action name="actionMedian_Filter">
   <property name="text">
    <string>Median Filter</string>
   </property>
  </action>
  <action name="actionMax_Filter">
   <property name="text">
    <string>Maximum Filtering</string>
   </property>
  </action>
  <action name="actionMinimum_Filtering">
   <property name="text">
    <string>Minimum Filtering</string>
   </property>
  </action>
  <action name="actionDFT_Smoothing_Image">
   <property name="text">
    <string>DFT Smoothing Image</string>
   </property>
  </action>
  <action name="actionDFT_Smoothing_Image_2">
   <property name="text">
    <string>DFT Smoothing Image 2</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionRoberts">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionRobert">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionRobert_2">
   <property name="text">
    <string>Roberts</string>
   </property>
  </action>
  <action name="actionDilasi">
   <property name="text">
    <string>Dilasi</string>
   </property>
  </action>
  <action name="actionErosi">
   <property name="text">
    <string>Erosi</string>
   </property>
  </action>
  <action name="actionOpening">
   <property name="text">
    <string>Opening</string>
   </property>
  </action>
  <action name="actionClosing">
   <property name="text">
    <string>Closing</string>
   </property>
  </action>
  <action name="actionDeteksi_Plat_Nomor">
   <property name="text">
    <string>Deteksi Plat Nomor</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSaveToFile">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
